resources/public/css/legacy.css

```diff
@@ -447,3 +447,29 @@ pre.highlight > code {
   border: none;
   color: inherit;
 }
+
+.day-arrows {
+  margin-left: 1rem;
+}
+
+.day-arrows .day-prev {
+    display: inline-block;
+    margin-right: 0.3rem;
+    background-color: rgb(62, 49, 60);
+    color: white;
+    min-height: 1rem;
+    min-width: 1rem;
+    text-align: center;
+    border-radius: 20%;
+}
+
+.day-arrows .day-next {
+    display: inline-block;
+    margin-left: 0.3rem;
+    background-color: rgb(62, 49, 60);
+    color: white;
+    min-height: 1rem;
+    min-width: 1rem;
+    text-align: center;
+    border-radius: 20%;
+}
```

Comment:
This style sheet is copied over from the old app, that's why it's called "legacy". I'd prefer to move the new stuff into Garden (`style.clj`). What do you think?

Reply:
Btw, do you want to just migrate the contents of legacy.css into style.clj then? Or is there is some plan where it makes more sense to move the rules as things are being added/modified?

Reply:
Yeah, I would prefer to move things there, but it also seems a lot of effort, unless it can be done automatically. I imagine that if an actual CSS person comes along they would want to redo much of it anyway with whatever style/layout framework they prefer, so that's why I didn't want to put too much effort into it yet.